x11rb/src/protocol/bigreq.rs

```diff
@@ -4,6 +4,7 @@
 //! Bindings to the `BigRequests` X11 extension.
 
 #![allow(clippy::too_many_arguments)]
+#![allow(clippy::unnecessary_to_owned)]
```

Comment:
Can you tell us something about this? In `x11rb/`, this seems definitely unnecessary since that's the only change in this crate, but I also cannot figure out what is "going on" in `x11rb-protocol/` with clippy and this lint.

Reply:
I use `cloned()` in certain cases because it's necessary for `u32::from()` which expects a value instead of a reference, but I figured out an optimization that lets me avoid needlessly using `cloned()` so I'll just delete this part.